### PR TITLE
Add ability to update target gas price.

### PR DESCRIPTION
**Pull request by bgravenorst**
## Describe the change

Add an API that updates the target gas price while Besu is running.

### CODE REVIEW

File: docs/Reference/API-Methods.md [hunk 1]
```diff
@@ -4484,6 +4484,43 @@ Returns full trace of all invoked opcodes of all transactions included in the bl
     methods, use the [`--rpc-http-api`](CLI/CLI-Syntax.md#rpc-http-api) or
     [`--rpc-ws-api`](CLI/CLI-Syntax.md#rpc-ws-api) options.
 
+### miner_changeTargetGasLimit
+
+Updates the target gas limit set using the [`--target-gas-limit`](CLI/CLI-Syntax.md#target-gas-limit)
```
Comment (NicolasMassart) [inline docs/Reference/API-Methods.md:4489]: Sentence looks strange... I had to read it multiple times to finally understand...

```suggestion
Updates the target gas limit that was set using the [`--target-gas-limit`](CLI/CLI-Syntax.md#target-gas-limit)
```

File: docs/Reference/API-Methods.md [hunk 2]
```diff
@@ -4484,6 +4484,43 @@ Returns full trace of all invoked opcodes of all transactions included in the bl
     methods, use the [`--rpc-http-api`](CLI/CLI-Syntax.md#rpc-http-api) or
     [`--rpc-ws-api`](CLI/CLI-Syntax.md#rpc-ws-api) options.
 
+### miner_changeTargetGasLimit
```
Comment (NicolasMassart) [inline docs/Reference/API-Methods.md:4487]: What if no `--target-gas-limit` CLI was used before, what is the response of this method?

Comment (bgravenorst) [reply docs/Reference/API-Methods.md:4487]: When I tested this I received an error stating no target gas limit has been set.

**Review by NicolasMassart — APPROVED** [review]

File: docs/Reference/API-Methods.md (hunk 2 above)
Comment (bgravenorst) [reply docs/Reference/API-Methods.md:4581]: The error told me exactly what the issue was, so it was clear enough.

**Pull request merged**